website/docs/r/cdn_domain.html.markdown

```diff
@@ -98,29 +98,34 @@ The config supports the following:
-    
+
 * `auth_config` - (Optional, Type: set)  Auth config of the accelerated domain. It's a set and consist of at most 1 item.
     * `auth_type` - (Optional) Auth type of the auth config. Valid values are  `no_auth`, `type_a`, `type_b` and `type_c`. Default value is `no_auth`.
     * `master_key` - (Optional) Master authentication key of the auth config. This parameter can have a string of 6 to 32 characters and must contain only alphanumeric characters.
     * `slave_key` - (Optional) Slave authentication key of the auth config. This parameter can have a string of 6 to 32 characters and must contain only alphanumeric characters.
     * `timeout` - (Optional, Type: int)  Authentication cache time of the auth config. Default value is `1800`. It's value is valid only when the `auth_type` is `type_b` or `type_c`.
-    
+
+* `https_config` - (Optional, Type: set)  Https config of the accelerated domain. It's a set and consist of at most 1 item.
+    * `server_certificate_status` - (Optional) This parameter indicates whether or not enable https. Valid values are `on` and `off`. Default value is `on`.
+    * `server_certificate` - (Optional) The SSL server certificate string. This is required if `server_certificate_status` is `on`
+    * `private_key` - (Optional) The SSL private key. This is required if `server_certificate_status` is `on`
+
 * `http_header_config` - (Optional, Type: set) Http header config of the accelerated domain. It's a set and consist of at most 8 items. The `header_key` for each item can not be repeated.
     * `header_key` - (Required) Header key of the http header. Valid values are `Content-Type`, `Cache-Control`, `Content-Disposition`, `Content-Language`，`Expires`, `Access-Control-Allow-Origin`, `Access-Control-Allow-Methods` and `Access-Control-Max-Age`.
     * `header_value` - (Required) Header value of the http header.
-    
+
 * `cache_config` - (Optional, Type: set)  Cache config of the accelerated domain. It's a set and each item's `cache_content` can not be repeated.
     * `cache_type` - (Required) Cache type of the cache config. Valid values are `suffix` and `path`.
-    * `cache_content` - (Required) Cache content of the cache config. It's value is a path string when the `cache_type` is `path`. When the `cache_type` is `suffix`, it's value is a string which contains multiple file suffixes separated by commas. 
+    * `cache_content` - (Required) Cache content of the cache config. It's value is a path string when the `cache_type` is `path`. When the `cache_type` is `suffix`, it's value is a string which contains multiple file suffixes separated by commas.
     * `ttl` - (Required, Type: int) Cache time of the cache config.
     * `weight` - (Optional, Type: int) Weight of the cache config. This parameter's value is between 1 and 99. Default value is `1`. The higher the value, the higher the priority.
 
```

Comment:
Can you improve this docs by using `block` to arrange several configs?

Reply:
I'm sorry I don't understand, what does it mean to use `block` to arrange configs? Is there an example in another part of the docs I can use as a reference? Sorry, I'm new to this

Reply:
Please refer to: https://github.com/terraform-providers/terraform-provider-alicloud/blob/master/website/docs/r/oss_bucket.html.markdown#block-website